### PR TITLE
How to add docs section

### LINKED ISSUE #756: [DOCS] How to add documentation
## 📚 Documentation

How to add documentation section in cursOfMyrra docs

### PR DESCRIPTION
**Pull request by BertovDev**
Add a section to the docs on how to add new documentation to it.

closes #756

### CODE REVIEW

File: docs/src/add_documentation.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+## Install mdbook
+
+☝️🤓 First make sure you have installed mdbook in your machine. Run `mdbook -V`. If you have it you will see something like `mdbook v0.4.34` if you dont, install it with the following:
+
+```
+cargo install mdbook
+cargo install mdbook-mermaid
+```
+
+Now you are ready to continue , for more information check the mdbook [mdbook docs](https://rust-lang.github.io/mdBook/).
```
Comment (manucamejo) [inline docs/src/add_documentation.md:12]: Remove the space before the comma.

File: docs/src/add_documentation.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Create a new section
+
+If you want to add a new documentation page it is pretty simple , follow the steps bellow
```
Comment (manucamejo) [inline docs/src/add_documentation.md:16]: And here

File: docs/src/add_documentation.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+### Using the Command Line :
+
+In the root of the project type:
+
+```
+cd docs/src
+```
+
+Inside the `src` folder create your markdown document ( without curly braces ).
```
Comment (manucamejo) [inline docs/src/add_documentation.md:26]: Here, after the ( and before the )

**Review by manucamejo — DISMISSED** [review]

File: docs/src/add_documentation.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# How to add documentation
+
+## Install mdbook
+
+☝️🤓 First make sure you have installed mdbook in your machine. Run `mdbook -V`. If you have it you will see something like `mdbook v0.4.34` if you don't, install it with the following:
```
Comment (tvillegas98) [inline docs/src/add_documentation.md:5]: with the following command lines

**Review by manucamejo — APPROVED** [review]

**Review by saugon — APPROVED** [review]

**Pull request merged**